README.rst

```diff
@@ -14,23 +14,122 @@ Lasagne
 =======
 
 Lasagne is a lightweight library to build and train neural networks in Theano.
+Its main features are:
 
-For support, please refer to the lasagne-users mailing list: https://groups.google.com/forum/#!forum/lasagne-users
+* Supports feed-forward networks such as Convolutional Neural Networks (CNNs),
+  recurrent networks including Long Short-Term Memory (LSTM) [coming soon!],
+  and any combination thereof
+* Allows architectures of multiple inputs and multiple outputs, including
+  auxiliary classifiers
+* Many optimization methods including Nesterov momentum, RMSprop and ADAM
+* Freely definable cost function and no need to derive gradients due to
+  Theano's symbolic differentiation
+* Transparent support of CPUs and GPUs due to Theano's expression compiler
 
-Documentation: http://lasagne.readthedocs.org/ (work in progress)
+Its design is governed by `six principles
+<http://lasagne.readthedocs.org/en/latest/user/development.html#philosophy>`_:
 
-Lasagne is a work in progress, input is welcome.
+* **Simplicity**: Be easy to use and easy to extend, with as few abstractions
+  as possible, to facilitate research
+* **Small interfaces**: Have as few classes and methods as possible
```

Comment:
This is very closely related to "Simplicity". In the long version, it even says "This should make it easier to both use the library and extend it". I'd merge it with Simplicity or drop it.